Sources/Private/CoreAnimation/CoreAnimationLayer.swift

```diff
@@ -331,7 +331,7 @@ extension CoreAnimationLayer: RootAnimationLayer {
   var isAnimationPlaying: Bool? {
     switch playbackState {
     case .playing:
-      return animation(forKey: #keyPath(animationProgress)) != nil
+      return pendingAnimationConfiguration?.playbackState == .playing || animation(forKey: #keyPath(animationProgress)) != nil
```

Comment:
Why does this `pendingAnimationConfiguration?.playbackState == .playing` check need to be nested in the `playbackState == .playing` branch? 

What happens if `pendingAnimationConfiguration?.playbackState == .playing` when `playbackState == .paused` or `nil`? Is there a reason we only have to do this check when another animation is already playing?

Reply:
That's a good point, I guess I was too focused on that this is the line that changed and therefore I wanted to put it there. I'll push a different proposal and you can react to that :-D

Reply:
Updates lgtm, thanks!